src/developer/web-api/tracker.md

```diff
@@ -1492,6 +1500,48 @@ Two endpoints are dedicated to tracked entities:
 - `GET /api/tracker/trackedEntities/{id}`
   - retrieves a tracked entity given the provided id
 
+#### Response content types
+
+If not otherwise specified, JSON is the default response for the `GET` method. 
+The API also supports CSV export for single and collection endpoints. Furthermore, compressed CSV types is an option for the collection endpoint.
+
+###### CSV
+
+In the case of CSV, the `fields` request parameter has no effect, and the response will always contain the following fields:
+
+  - trackedEntity (UID)
+  - trackedEntityType (UID)
+  - createdAt (Datetime)
+  - createdAtClient (Datetime)
+  - updatedAt (Datetime)
+  - updatedAtClient (Datetime)
+  - orgUnit (UID)
+  - inactive (boolean)
+  - deleted (boolean)
+  - potentialDuplicate (boolean)
+  - geometry (WKT, https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry. You can omit it in case of a `Point` type and with `latitude` and `longitude` provided)
+  - latitude (Latitude of a `Point` type of Geometry)
+  - longitude (Longitude of a `Point` type of Geometry)
+  - attribute (UID)
+  - displayName (String)
+  - attrCreatedAt (Attribute creation Datetime)
+  - attrUpdatedAt (Attribute last update Datetime)
+  - valueType (String)
+  - value (String)
+  - storedBy (String)
+  - createdBy (Username of user)
+  - updatedBy (Username of user)
+
+See [Tracked Entities](#tracked-entities) and [Attributes](#attributes) for more field descriptions.
+
+#### GZIP
+
+The response is a default `trackedEntities.csv.gz` file containing the `trackedEntities.csv` file.
```

Comment:
> is a default `trackedEntities.csv.gz` file

reads a bit odd (same for zip) 🤔 is it 

The response is file `trackedEntities.csv.gz` containing the `trackedEntities.csv` file.